app/assets/javascripts/controllers/ems_common/arbitration_profile_form_controller.js

```diff
@@ -79,9 +82,9 @@ ManageIQ.angular.app.controller('arbitrationProfileFormController', ['$scope', '
   var emsId = (/ems_cloud\/arbitration_profile_edit\/(\d+)/.exec($location.absUrl())[1]);
   var redirectUrl = '/ems_cloud/arbitration_profiles/' + emsId + '?db=ems_cloud';
 
-  var convertToString = function(id) {
+  var setRecordId = function(id) {
```

Comment:
@himdel, removed accidentally left console.log statements, i had to return ' ' from setRecordId function as having an undefined id was causing issue on the form, do not recall if it was due to form buttons enable/disable state or if there was another reason